Kitt-AI/snowboy
# Implemented setup.py that compiles python wrapper using Makefile

This should allow project to be more easily deployed to Pypi, solving #73 and simplifying the creation of projects that depend on snowboy. It installs a `snowboy` module with `snowboydecoder` and `snowboydetect` submodules to the python path.

To test it out, run:

```
python setup.py build
python setup.py install
``` 

Notice that description and authors fields were intentionally left blank for the maintainers to fill as they desire:

```
setup(
    name='snowboy',
    version='0.1',
    description='',
    maintainer='',
    maintainer_email='',
    license='',
    url='',
    packages=find_packages('examples/Python/'),
    package_dir={'snowboy': 'examples/Python/'},
    py_modules=['snowboy.snowboydecoder', 'snowboy.snowboydetect'],
    include_package_data=True,
    long_description="",
    classifiers=[],
    install_requires=[
        'PyAudio',
    ],
    cmdclass={
        'build': SnowboyBuild
    }
)
```
